### ISSUE TITLE
due:tomorrow on Windows app not working

### ISSUE THREAD

**Issue by csgrig73** [benrhughes/todotxt.net#413]
Hi there. New to Todotxt.net. Trying to filter tomorrow's tasks but the filter _"due:tomorrow"_ doesn't seem to work: I get no tasks at all. Tried the same filter with the Chrome extension (used the same todo file) and it worked fine.

Any ideas?

**Comment by remingtonsmith** [comment]
I don't believe the today, tomorrow, Monday, Tuesday, Wednesday, Thursday, Friday, Saturday, or Sunday due date shortcuts work within the filter settings.  It is just a shortcut for creating deadlines within tasks without having to look at a calendar.  When using it to create a task deadline, the software immediately translates it to an actual date once you finish entering the task.

**Comment by chipbite** [comment]
@remingtonsmith +1 
Adding `date shortcuts` could be done as a **new feature**. due:tomorrow could be useful for some, perhaps.
